jwcrypto/jwk.py

```diff
@@ -494,18 +494,53 @@ def add(self, elem):
             raise TypeError('Only JWK objects are valid elements')
         set.add(self, elem)
 
+
+class JWKSet(dict):
+    """A set of JWK objects.
+
+    Inherits from the standard 'dict' bultin type.
+    Creates a special key 'keys' that is of a type derived from 'set'
+    The 'keys' attribute accepts only :class:`jwcrypto.jwk.JWK` elements.
+    """
+    def __init__(self, *args, **kwargs):
+        dict.__init__(self)
+        dict.__setitem__(self, 'keys', _jwkset())
```

Comment:
Yeah, switching between languages makes you make the stupidest mistakes, revised patch pushed